exp-player/addon/components/exp-player.js

```diff
@@ -11,6 +11,21 @@ export default Ember.Component.extend(FullScreen, {
     session: null,
     frames: null,
 
+    store: Ember.inject.service(),
+    getCurrentUser: Ember.inject.service(),
+    pastSessions: Ember.computed('experiment', function() {
+        return this.get('getCurrentUser').load().then(([, profile]) => {
+            var profileId = profile.get('profileId');
+            var experimentId = this.get('experiment.id');
+            return this.get('store').queryRecord(this.get('experiment.sessionCollectionId'), {
+                q: [
+                    `experimentId:${experimentId}`,
+                    `profileId:${profileId}`
```

Comment:
Also, what is the rationale for putting store queries (for pastSessions) on the player, instead of on the route? It seems weird to have responsibility for prequisite data live in more than one place.